assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -65,25 +65,27 @@ export default function WidgetAreaRenderer( { slug } ) {
 
 	return (
 		<Grid className={ `googlesitekit-widget-area googlesitekit-widget-area--${ slug } googlesitekit-widget-area--${ style }` }>
-			<Row>
-				<Cell className="googlesitekit-widget-area-header" size={ 12 }>
-					{ Icon && (
-						<Icon width={ 33 } height={ 33 } />
-					) }
+			{ totalAreas > 1 && (
+				<Row>
+					<Cell className="googlesitekit-widget-area-header" size={ 12 }>
+						{ Icon && (
+							<Icon width={ 33 } height={ 33 } />
+						) }
```

Comment:
This would be good to add a unit test for. There's already test infrastructure for the `WidgetAreaRenderer`—could you add a test in https://github.com/google/site-kit-wp/blob/develop/assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js#L68 to ensure the title only appears with multiple areas?